Makefile

```diff
@@ -5,7 +5,7 @@ VERSION=$(shell git describe --tags --abbrev=0)
 all: docs compile checksum clean
 
 test:
-	/usr/bin/go test -timeout 30s github.com/terraform-routeros/terraform-provider-routeros
+	go test -timeout 30s github.com/terraform-routeros/terraform-provider-routeros
```

Comment:
I can happily change this. I don't have go installed on the same path and was giving me errors. The rest of the file is using just "go" instead of "/usr/bin/go"

Reply:
ok